docs/manage-validator/maxHealthChecks.md

```diff
@@ -0,0 +1,180 @@
+---
+id: maxHealthChecks
+title: Enhance Validator Reliability maxHealthChecks
```

Comment:
we should rename this title to reflect it as a feature instead of an announcement